src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -562,9 +577,52 @@ public void canCreateInstanceSourceRecord() throws Exception {
     assertThat(getResponse.getJson().size(), is(2));  // leader and fields
   }
 
+  @Test  // https://issues.folio.org/browse/MODINVSTOR-142?focusedCommentId=33665#comment-33665
+  public void canCreateInstanceSourceRecord101073931X() throws Exception {
+    UUID id = UUID.randomUUID();
+    JsonObject instance = smallAngryPlanet(id);
+    createInstance(instance);
+
+    put(id, toMarcJson("/101073931X.mrcjson"));
+
+    Response getResponse = getMarcJson(id);
+    assertThat(getResponse.getStatusCode(), is(200));
+    JsonArray fields = getResponse.getJson().getJsonArray("fields");
+    assertThat(fields.getJsonObject(0).getString("001"), is("101073931X"));
+  }
+
+  @Test  // https://issues.folio.org/browse/MODINVSTOR-143?focusedCommentId=33618#comment-33618
```

Comment:
This comment suggests that the change to use the PotgresClient upsert also resolved the single quote character error for one of the samples. Is that the case?

Reply:
I've verified this locally and have included the last sample file in the set of sample records to import